lua/presence/init.lua

```diff
@@ -502,16 +502,21 @@ function Presence:update_for_buffer(buffer, should_debounce)
         state = status_text,
         assets = assets,
         timestamps = {
-            start = activity_set_at,
+            start = relative_activity_set_at,
         },
     }
 
-    self.log:debug(string.format("Getting project name for %s...", parent_dirpath))
+    if not self.options.enable_line_number then
```

Comment:
Looks like this condition should not have the `not` since we'd be getting the line number when `self.options.enable_line_number` is `true`. In regards to my other comment below, we might also wanna move these debug log statements to within their respective `if` statements in the refactored code.